conjure-java-core/src/test/java/com/palantir/conjure/java/types/WireFormatTests.java

```diff
@@ -561,6 +567,52 @@ public void testOptionalAliasExample() throws IOException {
                 .isEqualTo(OptionalAliasExample.of(OptionalAlias.of(Optional.empty())));
     }
 
+    @Test
+    public void testEmptyOptionalFieldSerialization() throws IOException {
+        assertThat(mapper.writeValueAsString(PrimitiveOptionalsExample.builder()
+                        .aliasTwo(StringAliasTwo.of(Optional.empty()))
+                        .aliasOptional(OptionalAlias.of(Optional.empty()))
+                        .aliasOptionalMap(OptionalMapAliasExample.of(Optional.empty()))
```

Comment:
maybe we want tests for set/list as well for completeness?

Reply:
Updated :+1: